### PR TITLE
Install: Add Information About New Homebrew Formula

### PR DESCRIPTION
**Pull request by sanssecours**
# Purpose

This PR adds information about the new Homebrew formula for Elektra in [`homebrew-core`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/elektra.rb).

# Checklist

- [x] I checked all commit messages.
- [x] This PR adds documentation to `doc/INSTALL.md`.
- [x] I also updated the release notes.

### CODE REVIEW

File: doc/news/_preparation_next_release.md [hunk 1]
```diff
@@ -37,8 +37,6 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.8.
 - New Logo
 - INI as new default configuration file format
 - Bindings for Asynchronous I/O
-- <<HIGHLIGHT2>>
```
Comment (markus2330) [inline doc/news/_preparation_next_release.md]: I would prefer to keep these placeholders until short before the release. I hope it encourages people to add their contributions as highlight.

**Comment by markus2330** [comment]
Thank you for the PR, the fixes look great! Is this ready to merge?

**Comment by sanssecours** [comment]
> Is this ready to merge?

As far as I can tell, yes. I don’t think the changes in the PR should break anything in the macOS Travis build jobs.

**Comment by markus2330** [comment]
Thank you, great work!

**Pull request merged**